packages/calling-component-bindings/src/utils/videoGalleryUtils.ts

```diff
@@ -94,7 +94,7 @@ const memoizedAllConvertRemoteParticipant = memoizeFnAll(
     isMuted: boolean,
     isSpeaking: boolean,
     videoStreams: { [key: number]: RemoteVideoStreamState },
-    state: RemoteParticipantConnectionState,
+    state: RemoteParticipantConnectionState | 'Reconnecting',
```

Comment:
Should we just add this to our `RemoteParticipantConnectionState` instead of the union?